pyproject.toml

```
@@ -27,23 +27,41 @@ classifiers = [
 keywords = ["exoplanet", "atmosphere"]
 requires-python = '>=3.10'
 dependencies = [
-  'matplotlib',
-  'natsort',
-  'netcdf4',
-  'numpy',
-  'pandas',
-  'scipy',
-  'seaborn',
-  'tomlkit',
-  'f90nml' 
+  "matplotlib",
+  "natsort",
+  "netcdf4",
+  "numpy",
+  "pandas",
+  "scipy",
+  "seaborn",
+  "tomlkit",
+  "f90nml"
 ]
 
 [project.urls]
 homepage = "https://github.com/FormingWorlds/JANUS"
 
+[project.optional-dependencies]
+develop = [
+  "bump-my-version",
+  "pytest"
+]
+publishing = [
+  "wheel",
+  "build"
```

I suggest adding twine here, which is used for manual upload (in case the workflow fails).

```suggestion
  "twine",
  "wheel",
  "build"
```